add-ons/dynamic-admission-controllers/dynamic-admission-controllers.addon

```diff
@@ -0,0 +1,18 @@
+# Name: dynamic-admission-controllers
+# Description: Enables ValidatingAdmissionWebhook & MutatingAdmissionWebhook
+# Url: https://docs.openshift.com/container-platform/latest/architecture/additional_concepts/dynamic_admission_controllers.html
+# OpenShift-Version: >=3.10.0
+
+echo Enable required admission configs
+
+ssh sudo echo 's/"admissionConfig":{"pluginConfig":null}/"admissionConfig": {"pluginConfig": {"ValidatingAdmissionWebhook": {"configuration": {"apiVersion": "v1", "kind": "DefaultAdmissionConfig", "disable": false}}, "MutatingAdmissionWebhook": { "configuration": { "apiVersion": "v1", "kind": "DefaultAdmissionConfig", "disable": false }}}}/' > master-config.sed
+ssh sudo sed -i -f master-config.sed /var/lib/minishift/base/kube-apiserver/master-config.yaml
```

Comment:
@literalice  based on my testing we don't actually need to do this, this can be set on the openshift master-config.yaml and it works well.  I am not sure if we can use `minishift config patch set ..` from the add-on as that will help avoiding sed. @LalatenduMohanty @praveenkumar  thoughts ?

if thats not possible can you please use this patching to update the `/var/lib/minishift/base/openshift-apiserver/master-config.yaml` @LalatenduMohanty @praveenkumar is this the same file that will be updated by minishift config patch command ?

Reply:
@kameshsampath @literalice `minishift config patch set ..` this is something not able to run from the addon directly :(.

> is this the same file that will be updated by minishift config patch command ?

@kameshsampath yes this is the default what get updated put I think both `apiserver` need to be updated for it.

Reply:
> yes this is the default what get updated put I think both apiserver need to be updated for it.
no I tested and just doing minishift patch does the work

Reply:
@literalice  can you please check how I have done here https://gist.github.com/kameshsampath/06cd0491faf11add3e80a0e99f1af432  and  use the same logic in your PR ?

Reply:
Thank you for a example code.
I've applied the logic in my PR.